db/seeds.rb

```diff
@@ -3,8 +3,8 @@ def self.start
     seed = Seed.new
     seed.generate_categories
     seed.generate_businesses
+    seed.generate_business_items
     seed.generate_users
-    seed.generate_items
```

Comment:
was this name change simply to clarify the association between items and business?